spec/mailers/candidate_mailer_spec.rb

```diff
@@ -73,7 +73,7 @@
         'Your application is being considered',
         'heading' => 'Dear Bob',
         'working days the provider has to respond' => '10 working days',
-        'magic link to authenticate' => 'http://localhost:3000/candidate/authenticate?token=raw_token&u=encrypted_id'
```

Comment:
One consequence of this is that candidates who receive a magic link just before we deploy this and try to use it after will get a 404. This is a bit confusing. Would it make sense to redirect the (legacy) `authenticate` route to `/sign-in/expired`?

Reply:
Good idea, will add